h/templates/activity/search.html.jinja2

```diff
@@ -211,163 +211,161 @@
-                    formmethod="POST"
-                    name="group_leave"
-                    value="{{ group.pubid }}"
-                    data-confirm-message="Are you sure you want to leave the group &quot;{{ group.name }}&quot;?">
-                {% trans %}Leave this group{% endtrans %}
-            </button>
-          </li>
-        {% endif %}
-      </ul>
-    </section>
+    <h1 class="search-result-sidebar__title">{{ user.name }}</h1>
 
-    {{ tags_section() }}
-    {# if the group has a concept of members display members and join text #}
-    {% if group.members %} 
-      {{ group_users("Members", group.members, group.creator) }}
-      <h3 class="group-invite__title">
-        {% trans %}Invite new members{% endtrans %}
-      </h3>
-      Sharing the link lets people join this group:
-    {# if the group has no concept of members display moderators and share text #}
-    {% else %}
-      {{ group_users("Moderators", group.moderators, group.creator) }}
-      <h3 class="group-invite__title">
-        {% trans %}Share group{% endtrans %}
-      </h3>
-      Sharing the link lets people view this group:
+    <section class="search-result-sidebar__section">
+    {% if user.description %}
+      <div class="search-result-sidebar__div-group-info">
+      {% for paragraph in group.description.split('\n') %}
+        <p>{{ paragraph }}</p>
+      {% endfor %}
+      </div>
     {% endif %}
-    <div class="group-invite__container js-copy-button">
-      <input class="group-invite__input js-select-onfocus"
-             data-ref="input"
-             value="{{ group.url }}">
-      <button class="group-invite__clipboard-button"
-              data-ref="button"
-              title="Copy to clipboard">
-        {{ svg_icon('copy_to_clipboard', 'group-invite__clipboard-image') }}
-      </button>
-    </div>
-  {% endcall %}
-{% endmacro %}
-
-{% macro user_sidebar(user, stats) %}
-  {% call sidebar(user.name, user.description) %}
     <section class="search-result-sidebar__section">
-      <dl>
-        {% if stats.annotation_count %}
-          <dt class="search-result-sidebar__dt">
-            {% trans %}Annotations:{% endtrans %}
-          </dt>
-          <dd class="search-result-sidebar__dd">{{ stats.annotation_count }}</dd>
-        {% endif %}
-
-        <dt class="search-result-sidebar__dt">
-          {% trans %}Joined:{% endtrans %}
-        </dt>
-        <dd class="search-result-sidebar__dd">{{ user.registered_date }}</dd>
-
-        {%- if user.location %}
-          <dt class="search-result-sidebar__dt">
-            {% trans %}Location:{% endtrans %}
-          </dt>
-          <dd class="search-result-sidebar__dd">{{ user.location }}</dd>
-        {% endif -%}
-
-        {%- if user.uri %}
-          <dt class="search-result-sidebar__dt">
-            {% trans %}Link:{% endtrans %}
-          </dt>
-          <dd class="search-result-sidebar__dd">
+      <div class="search-result-sidebar__div-group-info">
```

Comment:
This styles all contain the term `group-info` but are also used for the user profile sidebar. This suggests to me that we should come up with a more generic (and shorter!) naming scheme for these components.

Reply:
I changed this to subsection -hope that's a more acceptable name here.